### DISCUSSION TITLE
Still missing 31 coins

### DISCUSSION THREAD

**Discussion by qc-world** [Chia-Network/chia-blockchain#7658] [Support]
Hi

After upgrading to 0.0.9 from 0.0.5, I lost 84 coins. 53 were air dropped couple of days ago but I'm still missing 31 of these.

Please airdrop these ASAP.

My wallet address is.

tsit1njq2fghte3lzgcn28ayuqc65uwv7ht9zjzglhsuuspe8t5fs2yes7x5rcv

Thank you

**Comment by keliew** [comment]
is that the SillyCoin?
